core/Pos/Arbitrary/Core.hs

```diff
@@ -443,13 +442,14 @@ instance Arbitrary G.GenesisCoreData where
                          <$> arbitrary
                          <*> pure (wordILen - a)
                          <*> arbitrary
-                , pure $ G.ExponentialStakes wordILen
+                , pure $ G.ExponentialStakes wordILen (Types.mkCoin 1024)
```

Comment:
I still don't understand anything. Why can't we use `choose (1, 1024)` instead of `1024`, for example?
I know that it's unlikely to matter, but who knows, maybe tests will always pass with 1024 and fail with 1023 (or 42).